src/connectors/wdr.ts

```diff
@@ -0,0 +1,36 @@
+export {};
+
+// Connector for the WDR radio stations (1Live, WDR2, WDR3, WDR4, WDR5, COSMO, etc.)
+
+Connector.playerSelector = '.wdrrCurrentChannels';
+
+Connector.artistTrackSelector = '.wdrrCurrentShowTitleTitle';
+
+Connector.isPlaying = () => {
```

Comment:
```suggestion
Connector.pauseButtonSelector = '#playCtrl.playing';
```
Does this work instead?

Reply:
Yes this is working perfectly fine. Thank you